scripts/src/javatests/org/oppia/android/scripts/common/BUILD.bazel

```diff
@@ -38,3 +38,14 @@ kt_jvm_test(
         "//third_party:com_google_truth_truth",
     ],
 )
+
+kt_jvm_test(
+    name = "RepositoryFileTest",
+    srcs = ["RepositoryFileTest.kt"],
+    deps = [
+        "//scripts/src/java/org/oppia/android/scripts/common:repository_file",
+        "//testing:assertion_helpers",
```

Comment:
Here & in other PRs: please omit unless you need ``assertThrows``.

Reply:
Done.